dude.py

```diff
@@ -52,6 +53,7 @@
     ["vertagt", "Liebe SiMo"],
     ["schluesselinhaber", "Liebe/r Bewohner/in des Fachschaftsraums"],
     ["finanzen", "Sehr geehrte Menschen mit Ahnung der vielen Goldbarren"]
+    ["vorkurs", "Lieber AK Vorkurs"],
```

Comment:
A dict would `{"user": "greeting"}` would probably be a more appropiate data structure here.